src/50_Samples_%26_Templates/Product.html

```diff
@@ -249,53 +311,93 @@
        </li>
      </ul>
-    <amp-carousel id="product-carousel" width="1024" height="682" layout="responsive" type="slides">
-        <amp-img src="/img/product1_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
+
+    <amp-carousel id="green-apple-carousel" width="1024" height="682" layout="responsive" type="slides" [slide]="selectedSlide" on="goToSlide:AMP.setState(selectedSlide=event.index, selectedCarousel=green)" [class]="selectedCarousel == 'green' ? 'show' : 'hide'">
+         <amp-img src="/img/green_apple_1_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
+         <amp-img src="/img/green_apple_2_1024x685.jpg" width="1024" height="682" layout="responsive"></amp-img>
+    </amp-carousel>
+    <amp-carousel id="golden-apple-carousel" width="1024" height="682" layout="responsive" type="slides" [slide]="selectedSlide" on="goToSlide:AMP.setState(selectedSlide=event.index, selectedCarousel=golden)" class="hide" [class]="selectedCarousel == 'golden' ? 'show' : 'hide'">
         <amp-img src="/img/product1_alt1_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
-        <amp-img src="/img/product1_alt2_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
     </amp-carousel>
+    <amp-carousel id="red-apple-carousel" width="1024" height="682" layout="responsive" type="slides" [slide]="selectedSlide" on="goToSlide:AMP.setState(selectedSlide=event.index, selectedCarousel=red)" class="hide" [class]="selectedCarousel == 'red' ? 'show' : 'hide'">
+        <amp-img src="/img/red_apple_1_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
+        <amp-img src="/img/red_apple_2_1024x793.jpg" width="1024" height="682" layout="responsive"></amp-img>
+    </amp-carousel>
+
   </div>
 
-    <h5 class="price-description">Price: $1.99</h5>
     <!-- #### Product Configuration -->
-    <!-- The add to cart action is implemented using  [amp-form](/components/amp-form/). In our sample we use the [amp-selector](/components/amp-selector/) for selecting different product properties. Pressing the `ADD TO CART` button adds the product to a shopping cart page using the properties you have selected. Notice that the button URL contains the query `clientId={{ClientId}}`. If an item is successfully added to the shopping cart, we redirect the user to the shopping cart page. The redirect target is configured via a header in the server's form response ([redirecting after submission](https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/amp-form.md#redirecting-after-a-submission)):
+    <!-- We are using [amp-bind](https://www.ampproject.org/docs/reference/components/amp-bind) to update the price of the product depending on the colour
+      selected. We configure the price for each colour in the `amp-state` component.
+      We bind to the `text` attribute: we specify that the value of text is "Price:"" concatented with the
```

Comment:
please see my above comment: we could put the whole string in the json instead of splitting the price number and the string "Price:", is this what you meant